charts/light-beaconchain-explorer/README.md.gotmpl

```diff
@@ -0,0 +1,39 @@
+
+{{ template "chart.header" . }}
+{{ template "chart.deprecationWarning" . }}
+
+{{ template "chart.versionBadge" . }}{{ template "chart.typeBadge" . }}
+
+{{ template "chart.description" . }}
+
+{{ template "chart.homepageLine" . }}
+
+{{ template "chart.sourcesSection" . }}
+
+{{ template "chart.requirementsSection" . }}
+
+{{ template "chart.valuesSection" . }}
+
+# Example
+
+Usage: light-beaconchain-explorer -config config.yaml
+
+Helper:
+```shell
+Usage of ./explorer_linux_amd64:
+-config string
+    Path to the config file, if empty string defaults will be used
+```
+
+In order to name validators based on ranges the following file format can be provided YAML:
+```shell
+0-500 Nethermind lighthouse-0
```

Comment:
I think you're missing some `:`. in this example. And maybe use`yaml` instead of `shell` for the right coloring.

Reply:
Fixed.